src/ListGroup/ListGroup.js

```diff
@@ -0,0 +1,2 @@
+import { ListGroup } from 'react-bootstrap'
+export default ListGroup
```

Comment:
IIUC It should be possible to simply just re-export this in src/index.js
```export { ListGroup } from 'react-bootstrap'```

Reply:
do we not prefer to break these out into their own individual stories and give PatternFly design examples (like in #74)? I think this is a good discussion point here on what we want to do with these kind of components that we are inheriting/exporting directly. My general thought is that it is good to detail them in stories if we can, but I'm not opposed to just exporting them.

Reply:
The PF designers are used to seeing all of the components demo'd like we have in the test pages... thus the question ;)

https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/

Reply:
I believe this also follows the existing approach with MenuItem so if we change it, we should go back and update it too?